man/systemd.netservice.xml

```diff
@@ -0,0 +1,168 @@
+    </para>
+
+    <para>The main network service file must have the extension <filename>.netservice</filename>; other
+    extensions are ignored.</para>
+
+    <para>The <filename>.netservice</filename> files are read from the files located in the system
+    network directory <filename>/usr/lib/systemd/resolve</filename>, the volatile runtime network
+    directory <filename>/run/systemd/resolve</filename> and the local administration network
+    directory <filename>/etc/systemd/resolve</filename>. All configuration files are collectively
+    sorted and processed in lexical order, regardless of the directories in which they live.
+    However, files with identical filenames replace each other. Files in <filename>/etc</filename>
+    have the highest priority, files in <filename>/run</filename> take precedence over files with
+    the same name in <filename>/usr/lib</filename>. This can be used to override a system-supplied
+    configuration file with a local file if needed.</para>
+
+    <para>Along with the network service file <filename>foo.netservice</filename>, a "drop-in" directory
+    <filename>foo.netservice.d/</filename> may exist. All files with the suffix
+    <literal>.conf</literal> from this directory will be parsed after the file itself is
+    parsed. This is useful to alter or add configuration settings, without having to modify the main
+    configuration file. Each drop-in file must have appropriate section headers.</para>
+
+    <para>In addition to <filename>/etc/systemd/resolve</filename>, drop-in <literal>.d</literal>
+    directories can be placed in <filename>/usr/lib/systemd/resolve</filename> or
+    <filename>/run/systemd/resolve</filename> directories. Drop-in files in
+    <filename>/etc</filename> take precedence over those in <filename>/run</filename> which in turn
+    take precedence over those in <filename>/usr/lib</filename>. Drop-in files under any of these
+    directories take precedence over the main network service file wherever located. (Of course, since
+    <filename>/run</filename> is temporary and <filename>/usr/lib</filename> is for vendors, it is
+    unlikely drop-ins should be used in either of those places.)</para>
+  </refsect1>
+
+  <refsect1>
+    <title>[Service] Section Options</title>
+
+      <para>The network service file contains a <literal>[Service]</literal>
+      section, which specifies a discoverable network service announced in a
+      local network with Multicast DNS broadcasts.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>Name=</varname></term>
+          <listitem>
+            <para>An instance name of the network service as defined in <ulink
+            url="https://tools.ietf.org/html/rfc6763">RFC 6763</ulink>, e.g. "webserver".
+            If the instance name contains a substring "%h" then the substring is
```

Comment:
Hmm, is %h the only escape supported?

Reply:
Basically I tried to replicate the functionality described in `man avahi.service`. It supports only `%h`. Same goes to `_http._tcp` network types.